static/app/views/issueDetails/groupEventCarousel.tsx

```diff
@@ -337,20 +378,26 @@ export function GroupEventCarousel({event, group, projectSlug}: GroupEventCarous
           ]}
         />
         {xlargeViewport && (
-          <Button size={BUTTON_SIZE} onClick={copyLink}>
-            Copy Link
-          </Button>
+          <Tooltip title={t('Copy link to this issue event')}>
```

Comment:
Can we show the old UI while the feature flag is turned off? I think it makes sense to release the new icon buttons along with the dropdown.